releases/v0.6.0.md

```diff
@@ -112,6 +112,10 @@ There is also a new app:
 - API docs that are actually up to date, using typedoc! You can find them in (by @leaverou with help from @MysteryBlokHed in #498 #497 #549)
 - Updated color space diagram in https://colorjs.io/docs/spaces which is now dynamically generated via https://d2lang.com/ (by @leaverou)
 
+### Bug fixes
+
+- Object-oriented functions now work between different sources of Color.js (#605)
```

Comment:
```suggestion
- Object-oriented functions now work between different sources of Color.js (by @MysteryBlokHed in #605)
```

Reply:
Pretty sure there's an existing section on bugfixes btw

Reply:
I tried to Ctrl+F for "bug" or "fix" but didn't find anything (besides the list of PR's that still need to be added to the log)